styles/styles.css

```diff
@@ -241,3 +243,24 @@ main .section {
 main .section.highlight {
   background-color: var(--highlight-background-color);
 }
+
+#coming-back-summer-of-2022{
```

Comment:
no, too specific. You need to target an element that doesn't change when the text changes.

Reply:
@helms-charity  - Yes, fixed that via 

main > .section > .default-content-wrapper > h1:first-of-type